operators/src/processing/raster_vector_join/non_aggregated.rs

```diff
@@ -258,15 +258,15 @@ where
         for (raster_processor, new_column_name) in
             self.raster_processors.iter().zip(&self.column_names)
         {
+            // TODO: spawn task
             stream = Self::process_collections(
```

Comment:
why not spawn a task?

Reply:
it wasn't spawned in the first place and only discovered by clippy. I tried hacking it in, but it's not that easy because of lifetimes, so I guess this shouldn't be part of a PR that updates dependencies.